README.md

```diff
@@ -39,44 +39,74 @@ response = client.request(SentimentAnalysisRequest(text=input_text))
 
 For more examples, see the [examples](examples/examples.py) folder.
 
-## Contributing
 
-We welcome contributions to the project! To contribute, please follow these steps:
+# Contributing to Supercontrast
```

Comment:
if we are adding a contributing template, lets reference that from the readme rather than posting it twice (once in the readme once in the contributing template).